Tasks/Kubernetes/Tests/L0.ts

```diff
@@ -0,0 +1,390 @@
+import * as path from 'path';
+import * as assert from 'assert';
+import * as ttm from 'vsts-task-lib/mock-test';
+import tl = require('vsts-task-lib');
+import * as shared from './TestShared';
+
+describe('Kubernetes Suite', function() {
+    this.timeout(30000);
+    before((done) => {
+        process.env[shared.TestEnvVars.operatingSystem] = tl.osType().match(/^Win/) ? shared.OperatingSystems.Windows : shared.OperatingSystems.Other;
+        done();
+    });
+    beforeEach(() => {
+        process.env[shared.isKubectlPresentOnMachine] = "true";
+        delete process.env[shared.TestEnvVars.command];
+        delete process.env[shared.TestEnvVars.containerType];
+        delete process.env[shared.TestEnvVars.versionOrLocation];
+        delete process.env[shared.TestEnvVars.specifyLocation];
+        delete process.env[shared.TestEnvVars.versionSpec];
+        delete process.env[shared.TestEnvVars.checkLatest];
+        delete process.env[shared.TestEnvVars.namespace];
+        delete process.env[shared.TestEnvVars.arguments];
+        delete process.env[shared.TestEnvVars.useConfigurationFile];
+        delete process.env[shared.TestEnvVars.secretName];
+        delete process.env[shared.TestEnvVars.forceUpdate];
+        delete process.env[shared.TestEnvVars.outputFormat];
+        delete process.env[shared.TestEnvVars.kubectlOutput];
+    });
+    after(function () {
+    });
+
+    it('Run sucessfully when the user provides a specific location for kubectl', (done:MochaDone) => {
```

Comment:
type in sucessfully => successfully in every test.

Reply:
done